Node JS/session fswd 01/UI/food-adda/src/components/SearchNav.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import { Form } from 'react-bootstrap'
+
+export const SearchNav = () => {
+  return (
+    <div className='w-100' id='searchBar'>
+        {/* Search Bar  */}
+        <Form>
+            <Form.Control className=' bg-transparent border-bottom border-0' type="text" placeholder="Search" />
```

Comment:
it will accept a callback from props, and we will call it with the typed text. Also there is no save button.
Lastly there is a css issue, where I can see black text on black search box.